core/src/main/java/com/graphhopper/GraphHopper.java

```diff
@@ -1131,6 +1127,9 @@ public void close() {
         if (ghStorage != null)
             ghStorage.close();
 
+        chGraphs.values().forEach(RoutingCHGraph::close);
+        landmarks.values().forEach(LandmarkStorage::close);
```

Comment:
So far we only called `ghStorage.close()` here which 'closed' the base graph and the CH graphs, but not LM! Now we explicitly close CH and LM.